### DISCUSSION TITLE
Copilot always hangs at giving an example and never genrates the code: IntelliJ IDE

### DISCUSSION THREAD

**Discussion by viniteps** [community/community#124057] [Copilot Conversations]
### Select Topic Area

Bug

### Body

It seems that whenever I ask a question in the chat,[using IntelliJ IDEA 2022.2.5 (Ultimate Edition) ]
![copilot](https://github.com/community/community/assets/126535895/16c4f2b6-e925-438a-a053-53a7b891b427)
provides the answer but then gets stuck at "Here is an example:", without generating any examples. As a result, I have to cancel to ask anything else, but Copilot does not write code in the chat window.

**Comment by Akash1134** [comment]
Hi @viniteps ,

There are other more recent versions of your IDE [available](https://www.jetbrains.com/idea/download/other.html), update to the latest for both IntelliJ and Copilot if you haven't. 

Thanks!
